pkg/redis/client.go

```diff
@@ -533,3 +768,38 @@ func (c *redisClient) execute(ctx context.Context, wrappedCmd func() ErrCmder) (
 		return cmder, cmder.Err()
 	})
 }
+
+func (c *redisClient) toGosolineZs(zs []baseRedis.Z) []Z {
+	result := make([]Z, len(zs))
+	for i := range zs {
+		result[i] = Z{
+			Score:  zs[i].Score,
+			Member: zs[i].Member,
+		}
+	}
+
+	return result
+}
+
+func (c *redisClient) toGoRedisZs(zs []Z) []baseRedis.Z {
+	result := make([]baseRedis.Z, len(zs))
+	for i := range zs {
+		result[i] = baseRedis.Z{
+			Score:  zs[i].Score,
+			Member: zs[i].Member,
+		}
+	}
+
+	return result
+}
+
+func (c *redisClient) toGoRedisZAddArgs(args ZAddArgs, zs []baseRedis.Z) baseRedis.ZAddArgs {
+	return baseRedis.ZAddArgs{
+		NX:      args.NX,
+		XX:      args.XX,
+		LT:      args.LT,
+		GT:      args.GT,
+		Ch:      args.Ch,
+		Members: zs,
+	}
```

Comment:
```suggestion
	return baseRedis.ZAddArgs(args)
```

Reply:
Would love to do it that way, but `gosoline.redis.ZAddArgs` has the field`[]gosoline.Z Members` where `baseRedis.ZAddArgs` has the field`[]baseRedis.Z Members`